src/mgr/MgrPyModule.cc

```diff
@@ -171,31 +171,43 @@ int MgrPyModule::load()
   }
 
   // Find the class
-  // TODO: let them call it what they want instead of just 'Module'
-  auto pClass = PyObject_GetAttrString(pModule, (const char*)"Module");
-  Py_DECREF(pModule);
-  if (pClass == nullptr) {
-    derr << "Class not found in module '" << module_name << "'" << dendl;
+  auto mgr_module_type = PyImport_ImportModule("mgr_module.MgrModule");
```

Comment:
please note, `MgrModule` is not a "module". so we will always have following error when loading a mgr plugin:
```
2017-10-14 02:57:06.987 7f9448fbf700  1 mgr init Loading python module 'restful'
2017-10-14 02:57:06.987 7f9448fbf700 20 mgr Gil GIL acquired for thread state 0x5627fd09c9a0
2017-10-14 02:57:07.009 7f9448fbf700 20 mgr ~Gil GIL released for thread state 0x5627fd09c9a0
2017-10-14 02:57:07.009 7f9448fbf700 20 mgr Gil GIL acquired for thread state 0x5627fd09cbb0
2017-10-14 02:57:07.360 7f9448fbf700 -1 mgr load Unable to import MgrModule from mgr_module
2017-10-14 02:57:07.360 7f9448fbf700 -1 mgr load ImportError: No module named MgrModule
```
 @bhavishyagopesh  how did you test this change?

Reply:
instead, we need to `PyImport_ImportModule()` and then `PyObject_GetAttrString()`.